player/external_files.c

```diff
@@ -43,32 +43,48 @@ static const char *const audio_exts[] = {"mp3", "aac", "mka", "dts", "flac",
                                          NULL};
 
 static const char *const image_exts[] = {"jpg", "jpeg", "png", "gif", "bmp",
-                                         "webp", "jxl", "tiff", "tif",
+                                         "webp", "jxl", "tiff", "tif", "avif",
                                          NULL};
 
 // Stolen from: vlc/-/blob/master/modules/meta_engine/folder.c#L40
 // sorted by priority (descending)
 static const char *const cover_files[] = {
     "AlbumArt.jpg",
     "AlbumArt.webp",
+    "AlbumArt.jxl",
+    "AlbumArt.avif",
     "Album.jpg",
+    "Album.jxl",
     "Album.webp",
+    "Album.avif",
```

Comment:
The order should remain consistent here. i.e. `.jpg`, then `.webp`, `.jxl`, and `.avif`. Applies elsewhere of course.

Reply:
Should I change the first arrange from .jpg, then .webp, .jxl and .avif, if possible?

Reply:
I just meant that within the same basename, the order should be the same. So it should go: `Album.jpg`, `Album.webp`, `Album.jxl`, `Album.avif`. The same thing applies to `cover`, `front`, etc.

But honestly I don't really like this hardcoded list in the first place. What if my file has a `.jpeg` extension? It'll be completely ignored since it's not in this list.

Reply:
> I just meant that within the same basename, the order should be the same. So it should go: Album.jpg, Album.webp, Album.jxl, Album.avif. The same thing applies to cover, front, etc.

I will do arrange from JPG, then webp, jxl and avif.

> What if my file has a .jpeg extension? It'll be completely ignored since it's not in this list.

Yes, the .JPEG file extension completely ignores, if the external_files.c does not have .JPEG list on it. It needs be done manually detect for using CLI command `--cover-art-file` or mpv config file to detect FFmpeg image supported formats.

- Martin Eesmaa

Reply:
Screenshot proof:
![testedscreenshotofmpvbuildmyown](https://user-images.githubusercontent.com/88035011/220252911-f95a4097-5c75-4901-b7c6-5e204b9f2ecc.png)

- Martin Eesmaa

Reply:
I'm saying that webp should come before jxl.

Reply:
Ok, I will fix the code changes.